app/controllers/auth_controller.rb

```diff
@@ -16,6 +16,16 @@ def callback
     render json: result
   end
 
+  def accept_policy
+    return unless current_user?
```

Comment:
What is this intended to do? Returning like this won't send an appropriate response back, I think.

Also, doesn't seem like the accepting of the privacy policy belongs in the auth controller or `authentication` concern – it should probably have it's own endpoint + service, or alternatively could be a part of the new profile resource in https://github.com/chaynHQ/soulmedicine/pull/32.